api/types/types.go

```diff
@@ -86,6 +86,11 @@ var ControllerEditPolicyRule = []rbacv1.PolicyRule{
 		APIGroups: []string{"apps"},
 		Resources: []string{"deployments", "statefulsets"},
 	},
+	{
```

Comment:
should the deploy-watch-*.yaml's under the site controller be updated for daemonsets as well?

Reply:
No, the site controller doesn't watch them, only the service-controller.